src/main/java/muramasa/antimatter/network/packets/GuiEventPacket.java

```diff
@@ -18,20 +17,20 @@
 
     private final GuiEvent event;
     private final BlockPos pos;
-    private final int dimension;
+    private final int dim;
```

Comment:
Is there a reason why a dim id needs to be sent on GuiEventPacket? `ctx.get().getSender()` should take care of it.

Reply:
Did you meant CompoundPacket may be? Because GuiEventPacket made for server.

Reply:
GuiEventPacket is client -> server, correct? Hence you can do `ctx.get().getSender().getServerWorld()` instead of requiring a dim -> DimensionType (int id obsoleted in 1.16) -> using ServerLifecycleHooks

Reply:
Oh I didn't see that method, I replaced it already. thanks